test/e2e/vars.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -e
+export E2E_TEST_IMAGE CUSTOMIZED_DOCKER_PIPELINE_IMAGE_REF_LOCALHOST CUSTOMIZED_DOCKER_PIPELINE_IMAGE_REF_CLUSTER
+
+E2E_TEST_IMAGE=quay.io/konflux-ci/e2e-tests:5de23fe3dd41985e6ea1039f053f4165a60a414c
```

Comment:
In order to keep this image up-to-date, I would add a task similar to [this one](https://github.com/konflux-ci/e2e-tests/blob/ecbc63890c6f756a3a2ca5b367a5d1eda8173e27/.tekton/konflux-e2e-tests-push.yaml#L417-L440) to create a PR against this repo each time e2e-tests `main` branch is updated.

Reply:
you can use renovate for that.